web/packages/teleport/src/Notifications/Notification.tsx

```diff
@@ -34,26 +35,45 @@ import Dialog, {
 } from 'design/Dialog';
 import { Theme } from 'design/theme/themes/types';
 
-import { Notification as NotificationType } from 'teleport/services/notifications';
+import {
+  Notification as NotificationType,
+  NotificationState,
+} from 'teleport/services/notifications';
 
 import { useTeleport } from '..';
 
 import { NotificationContent } from './notificationContentFactory';
+import useStickyClusterId from 'teleport/useStickyClusterId';
+import { useAsync } from 'shared/hooks/useAsync';
 
 export function Notification({
   notification,
 }: {
   notification: NotificationType;
 }) {
   const ctx = useTeleport();
+  const { clusterId } = useStickyClusterId();
 
   const content = ctx.notificationContentFactory(notification);
 
-  // If the notification is unsupported, it should not be shown.
-  if (!content) {
-    return null;
-  }
+  const [markAsClickedAttempt, markAsClicked] = useAsync(() =>
+    ctx.notificationService
+      .upsertNotificationState(clusterId, {
+        notificationId: notification.id,
+        notificationState: NotificationState.CLICKED,
+      })
+      .then(res => {
+        notification.clicked = true;
```

Comment:
What does it do?
Mutating the state isn't a thing that we should do in React :(

Reply:
I can't remember why I did this, I think this was a leftover from testing something